api/envoy/config/core/v3/protocol.proto

```diff
@@ -345,6 +345,22 @@ message Http2ProtocolOptions {
   // <https://www.iana.org/assignments/http2-parameters/http2-parameters.xhtml#settings>`_ for
   // standardized identifiers.
   repeated SettingsParameter custom_settings_parameters = 13;
+
+  // Send HTTP/2 PING frames at this period, in order to test that the connection is still alive.
+  // If not specified, no keepalive PING frames will be sent.
```

Comment:
Should we have any built-in jitter here, either just implicit or defined as a param?

Reply:
Good point; we can probably borrow some of the stuff from health checks.

Reply:
+1